velox/functions/prestosql/aggregates/DecimalAggregate.h

```diff
@@ -232,6 +234,7 @@ class DecimalAggregate : public exec::Aggregate {
         auto serializedAccumulator =
             intermediateFlatVector->valueAt(decodedIndex);
         rows.applyToSelected([&](vector_size_t i) {
+          clearNull(group);
```

Comment:
This might be a naive question. In general it seems like we always want to clearNull except when the value is actually null. So why not always clearNull by default and only set null when it has to. Now we are calling clearNull in many code-paths that can be easy to miss.